environment.yml

```diff
@@ -98,19 +98,24 @@ dependencies:
   - zstd=1.3.7
   - pip:
     - absl-py==0.9.0
+    - aniso8601==8.0.0
     - astor==0.8.1
     - cachetools==4.0.0
     - chardet==3.0.4
+    - click==7.0
     - gast==0.2.2
     - google-auth==1.11.2
     - google-auth-oauthlib==0.4.1
     - google-pasta==0.1.8
     - grpcio==1.27.2
     - h5py==2.10.0
     - idna==2.9
+    - itsdangerous==1.1.0
+    - jinja2==2.11.1
     - keras-applications==1.0.8
     - keras-preprocessing==1.1.0
     - markdown==3.2.1
+    - markupsafe==1.1.1
```

Comment:
Can you add the packages you add to requirements.txt?

Reply:
Yeah.